source/MapPanel.cpp

```diff
@@ -991,14 +992,14 @@ void MapPanel::DrawWormholes()
 	
 	// Avoid iterating each StellarObject in every system by iterating over planets instead. A
 	// system can host more than one set of wormholes (e.g. Cardea), and some wormholes may even
-	// share a link vector. If a wormhole's planet has no description, no link will be drawn.
+	// share a link vector.
 	for(auto &&it : GameData::Planets())
 	{
```

Comment:
With a root-node approach, there will be a `GameData::Wormholes` collection you would iterate through here, rather than the planets collection.